setup.js

```diff
@@ -59,3 +65,42 @@ const WH = window.WH
 export {
     WH,
 }
+
+function firstOnApply (opts) {
+    var elements = opts.elements
+    var events = opts.events
+    var isDelegated = opts.isDelegated || false
```

Comment:
I would never use `var` but only `const` or `let`

Reply:
modified to const